src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -324,9 +343,93 @@ public Mapper(WordEmbeddingsTransform parent, Schema inputSchema)
                 _outputType = new VectorType(NumberType.R4, 3 * _parent._currentVocab.Dimension);
             }
 
+            public bool CanSaveOnnx(OnnxContext ctx) => true;
+
             public override Schema.Column[] GetOutputColumns()
                 => _parent.ColumnPairs.Select(x => new Schema.Column(x.output, _outputType, null)).ToArray();
 
+            public void SaveAsOnnx(OnnxContext ctx)
+            {
+                foreach (var (input, output) in _parent.Columns)
+                {
+                    var srcVariableName = ctx.GetVariableName(input);
+                    var schema = _parent.GetOutputSchema(InputSchema);
+                    schema.TryGetColumnIndex(output, out int colIndex);
+                    var dstVariableName = ctx.AddIntermediateVariable(schema.GetColumnType(colIndex), output);
```

Comment:
>schema.GetColumnType(colIndex) [](start = 70, length = 30)

schema[output].Type

Reply:
Fixed, thanks!

---
In reply to: [226402657](https://github.com/dotnet/machinelearning/pull/1249#discussion_r226402657) [](ancestors = 226402657)